tools-v2/pkg/cli/command/curvebs/list/chunkserver/chunkserver.go

```diff
@@ -0,0 +1,254 @@
+/*
+ *  Copyright (c) 2022 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: CurveCli
+ * Created Date: 2023-03-27
+ * Author: Sindweller
+ */
+
+package chunkserver
+
+import (
+	"context"
+	"fmt"
+	"github.com/dustin/go-humanize"
+	cmderror "github.com/opencurve/curve/tools-v2/internal/error"
+	cobrautil "github.com/opencurve/curve/tools-v2/internal/utils"
+	basecmd "github.com/opencurve/curve/tools-v2/pkg/cli/command"
+	"github.com/opencurve/curve/tools-v2/pkg/cli/command/curvebs/list/server"
+	"github.com/opencurve/curve/tools-v2/pkg/config"
+	"github.com/opencurve/curve/tools-v2/pkg/output"
+	"github.com/opencurve/curve/tools-v2/proto/proto/topology"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+	"strconv"
+)
+
+const (
+	chunkServerExample = `$ curve bs list chunkserver`
+)
+
+type ListChunkServerRpc struct {
+	Info                  *basecmd.Rpc
+	Request               *topology.ListChunkServerRequest
+	topologyServiceClient topology.TopologyServiceClient
+}
+
+var _ basecmd.RpcFunc = (*ListChunkServerRpc)(nil) // check interface
+
+type ChunkServerCommand struct {
+	basecmd.FinalCurveCmd
+	Rpc      []*ListChunkServerRpc
+	Response []*topology.ListChunkServerResponse
+}
```

Comment:
listchunkserver can be used independently as a command, and some [other pr](https://github.com/opencurve/curve/pull/2463) need its result.
just like [copyset.go](tools-v2/pkg/cli/command/curvebs/query/copyset/copyset.go).